sql-to-dbsp-compiler/lib/sqllib/src/operators.rs

```diff
@@ -0,0 +1,375 @@
+use dbsp::algebra::{F32, F64};
+
+use crate::for_all_compare;
+use crate::for_all_int;
+use crate::for_all_int_compare;
+use crate::for_all_numeric;
+use crate::for_all_numeric_compare;
+use crate::some_operator;
+
+#[inline(always)]
+pub fn eq_i16_i16(left: i16, right: i16) -> bool {
```

Comment:
You could generate most of these as well with macros. Maybe `eq_for` and so on.

Reply:
Do you know how to?
I need to paste the body of a function.

Reply:
Maybe by using a generic function